internal/generated/snippets/go.mod

```diff
@@ -1,6 +1,8 @@
 module cloud.google.com/go/internal/generated
 
-go 1.20
+go 1.21
+
+toolchain go1.21.13
```

Comment:
Just curious, why does this go.mod get `toolchain`, when others do not?

Reply:
good catch, I think I used my wrong tidy alias here. tidied again and gone.